apps/desktop-wallet/src/features/transactionsDisplay/transactionDetailsModal/TransactionDetailsModal.tsx

```diff
@@ -0,0 +1,151 @@
+/*
+Copyright 2018 - 2024 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { AddressHash } from '@alephium/shared'
+import { ALPH } from '@alephium/token-list'
+import { Transaction } from '@alephium/web3/dist/src/api/api-explorer'
+import { memo } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled, { useTheme } from 'styled-components'
+
+import ActionLink from '@/components/ActionLink'
+import Amount from '@/components/Amount'
+import Badge from '@/components/Badge'
+import DataList from '@/components/DataList'
+import Tooltip from '@/components/Tooltip'
+import { ModalBaseProp } from '@/features/modals/modalTypes'
+import AddressesDataRows from '@/features/transactionsDisplay/transactionDetailsModal/AddressesDataRows'
+import DirectionalInfo from '@/features/transactionsDisplay/transactionDetailsModal/DirectionalInfo'
+import FTAmounts from '@/features/transactionsDisplay/transactionDetailsModal/FTAmounts'
+import FTsDataListRow from '@/features/transactionsDisplay/transactionDetailsModal/FTsDataListRow'
+import GasUTXOsExpandableSection from '@/features/transactionsDisplay/transactionDetailsModal/GasUTXOsExpandableSection'
+import LockTimeDataListRow from '@/features/transactionsDisplay/transactionDetailsModal/LockTimeDataListRow'
+import NFTsDataListRow from '@/features/transactionsDisplay/transactionDetailsModal/NFTsDataListRow'
+import NSTsDataListRow from '@/features/transactionsDisplay/transactionDetailsModal/NSTsDataListRow'
+import TransactionType from '@/features/transactionsDisplay/transactionDetailsModal/TransactionType'
```

Comment:
I wonder if we should maybe create an `index.tsx` file in those feature folders, to allow for importing things in one line instead of this 🤔 WDYT?

Reply:
Not sure exactly what you mean, can you demonstrate? Will code navigation work fine if we do what you propose? (Cmd + Click on component name should navigate to the component file)

Reply:
Same as what we do in our libs, like in our shared lib: `src/store/index.ts`

```ts
export * from '@/store/store'
export * from '@/store/addresses'
export * from '@/store/prices'
export * from '@/store/assets'
export * from '@/store/network'
export * from '@/store/global'
export * from '@/store/settings'
export * from '@/store/clients'
```

Reply:
Ah, I see. Well, we could do that. But the LOC won't change because then we'd end up with

```ts
import {
 AddressesDataRows,
 DirectionalInfo,
 FTAmounts,
 FTsDataListRow,
 GasUTXOsExpandableSection,
 LockTimeDataListRow,
 NFTsDataListRow,
 NSTsDataListRow,
 TransactionType
} from '@/features/transactionsDisplay/transactionDetailsModal'
```

Then we need to always remember to maintain it. It's not rly a pattern we have in our apps, unlike our exportable packages.

Another thing, we tend to have `index.ts` fles to represent the actual component (like `AddressesPage/index.tsx`, but tbh I never liked that cause it doesn't help with file navigation so here I went for `transactionDetailsModal/TransactionDetailsModal.tsx`.

Anyway, I personally find it extra work for not much gain. WDYT?